src/utils/i18n.js

```diff
@@ -62,10 +62,22 @@ function hget(d, key, defaultValue){
 
 export function t(trans, lang, key, ...params){
     var kl = key
+    var fb
     if (!Array.isArray(kl))
         kl = [kl]
-    const value = hget(trans, [lang, ...kl])
-    if (value === undefined){
+    var value = hget(trans, [lang, ...kl])
+
+    // try to get first language as fallback
+    if (value === undefined) {
+        for (fb of trans.keys()) {
+            value = hget(trans, [fb, ...kl]);
+            if (value !== undefined) {
+                break;
+            }
+        }
+    }
```

Comment:
Can we instead call `t` with the first language as a parameter?
Also, Is this change related to the IE9 compatibility?

Reply:
I think not, do you really know, that 'en' for example is the first language? What if an sentance is missing in an language?

Reply:
Right, we shouldn't assume 'en' to be the default for the website owner's content.

Reply:
Perhabs it would be a good idea to use something like `defaults` as default language which must be filled with all needed strings. Then each translation is moved to config, so the base could be smaller. For me, but that's my personal opinion, it does not make sence to download an app with dozens of languages included. The other point why should I translate things, which must not.

Reply:
https://github.com/mkkeck/coi/commit/bf9bcd4cf5546bdebd8344c377c6ccb18fc7644b